site/lists/jobs.scroll

```diff
@@ -27,6 +27,8 @@ pipeTable
  Hubstaff|https://talent.hubstaff.com/
  Berlinstartupjobs|https://berlinstartupjobs.com/
  Vanhack|https://vanhack.com/jobs
+  Braintrust|https://www.usebraintrust.com/
```

Comment:
extra space

Reply:
<img width="351" alt="Screenshot 2022-10-04 at 17 23 36" src="https://user-images.githubusercontent.com/54355787/193845545-7cb4349d-dc53-4814-92ac-57ea151b842f.png">

Reply:
@breck7  I have made the changes

Reply:
great work!